### PR TITLE
services/bridge: use horizonclient

### PR DESCRIPTION
**Pull request by poliha**
This PR starts the process of swapping out the old sdk for the new one in the bridge server.
It focuses on replacing clients/horizon with clients/horizonclient.
We still need to replace the builder with txnbuild and implement some tests. The plan is to cover those in future PRs. 
This is part of the work required in #1251

### CODE REVIEW

File: services/bridge/internal/handlers/request_handler_reprocess.go [hunk 1]
```diff
@@ -30,13 +30,19 @@ func (rh *RequestHandler) Reprocess(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	operation, err := rh.Horizon.LoadOperation(request.OperationID)
+	operation, err := rh.Horizon.OperationDetail(request.OperationID)
```
Comment (bartekn) [inline services/bridge/internal/handlers/request_handler_reprocess.go]: Not in a scope of this PR but why `OperationDetail` requires operation id but `AccountDetail` requires `AccountRequest` struct?

File: services/bridge/internal/listener/payment_listener.go [hunk 2]
```diff
@@ -380,3 +369,65 @@ func (pl *PaymentListener) getMAC(key string, raw []byte) ([]byte, error) {
...
+				if ok {
+					payment.Amount = ace.Amount
+				} else {
+					return bridge.PaymentResponse{}, errors.New("unable to get amount from effect")
+				}
+			}
+		}
+
+	default:
+		return bridge.PaymentResponse{}, errors.New("operation not sent to configured receiving account")
```
Comment (bartekn) [inline services/bridge/internal/listener/payment_listener.go]: Error message should say that operation type is not permitted.

File: services/bridge/main.go [hunk 3]
```diff
@@ -137,10 +138,10 @@ func NewApp(config config.Config, migrateFlag bool, versionFlag bool, version st
 		Timeout: 60 * time.Second,
 	}
 
-	h := horizon.Client{
-		URL:     config.Horizon,
-		HTTP:    &httpClientWithTimeout,
-		AppName: "bridge-server",
+	h := hc.Client{
+		HorizonURL: config.Horizon,
+		HTTP:       http.DefaultClient,
```
Comment (bartekn) [inline services/bridge/main.go]: It should use `httpClientWithTimeout`.

File: services/bridge/internal/handlers/request_handler_reprocess.go (hunk 1 above)
Comment (poliha) [reply services/bridge/internal/handlers/request_handler_reprocess.go]: Yea I think this is something that we can update in the horizonclient package. We really dont need the whole request struct when getting the account detail.

File: services/bridge/main.go (hunk 3 above)
Comment (poliha) [reply services/bridge/main.go]: `horizonclient.Client` also has `horizonTimeOut` which defaults to 60s if it is not set. Do we want to set the timeout twice?

**Pull request merged**